src/components/modal/package.json

```diff
@@ -77,6 +77,56 @@
                         "description": "BvEvent object. Call bvEvt.preventDefault() to cancel hide"
                     }
                 ]
+            },
+            {
+                "event": "bv::show::modal",
+                "description": "Emitted on $root when modal is about to be shown.",
```

Comment:
Incorrect.  this is not emitted by modal at all.  But is used to control the modal, and should not be listed under this section.